### DISCUSSION TITLE
Crazy CPU usage from Kedro parameters.yml

### DISCUSSION THREAD

**Discussion by lucasjamar** [kedro-org/kedro#1013] [Idea]
Hi guys,

Has anyone experienced crazy CPU usage related to kedro?
It makes no sense... How can a tiny yaml take up so much CPU? If you have any clue what could be causing this, I would love to know, my computer has already refused to respond several times because of this. Thanks!

![image](https://user-images.githubusercontent.com/39372418/139841216-35a88cca-a5db-4189-a77e-fcabe15042a0.png)

**Comment by lucasjamar** [comment]
Sorry i meant memory not CPU

**Reply by datajoely** [reply]
Hi @lucasjamar are you seeing Kedro take up a lot of memory at run-time or is the question about the PyCharm editor taking up a lot of memory when it's in use?

**Comment by lucasjamar** [comment]
Sorry @datajoely , that was an obvious thing to double check. It is indeed a pycharm issue. I just thought it was kedro because it was pointing to the parameters.yml.

**Reply by datajoely** [reply]
I think in this case - it's just Windows taking the current window title i.e. your editor has this page open
